closed/make/modules/openjceplus/Copy.gmk

```diff
@@ -51,7 +51,7 @@ ifeq (true,$(BUILD_OPENJCEPLUS))
   $(eval $(call SetupCopyFiles, OPENJCEPLUS_OCK_COPY, \
       SRC := $(OPENJCEPLUS_OCK_DIR), \
       DEST := $(SUPPORT_OUTPUTDIR)/$(OPENJCEPLUS_OCK_SUB_DIR)/$(MODULE), \
-      FILES := $(call FindFiles, $(OPENJCEPLUS_OCK_DIR)), \
+      FILES := $(filter %.dll %.dylib %.so %.x %.txt, $(call FindFiles, $(OPENJCEPLUS_OCK_DIR))), \
```

Comment:
Suggest we should have a variable for this pattern:
```
      FILES := $(filter $(OPENJCEPLUS_FILTER_PATTERN) %.txt, $(call FindFiles, $(OPENJCEPLUS_OCK_DIR))), \
```
where `OPENJCEPLUS_FILTER_PATTERN` is defined above
```
  OPENJCEPLUS_FILTER_PATTERN := %$(SHARED_LIBRARY_SUFFIX) %.txt
  ifeq ($(call isTargetOs, zos), true)
    OPENJCEPLUS_FILTER_PATTERN += %.x
  endif
```
and also used on line 36.

Can you give some examples of files that would be omitted by this new filter?

Reply:
Perhaps just two uses don't fully support the introduction of a variable, but I do think it was a mistake that the pattern included elements that should not match (e.g. `%.dll` should only apply on Windows, `%.dylib` only on macOS).

I was hoping you could be more specific than "an unexpected file".

Reply:
We want to remove the `Standard_International_Program_License_Agreement.pdf` file that started showing up in the lib directory.

Reply:
I understand, but the filter should not match files that are not appropriate for the target platform.

If `.x` files are not needed (even on z/OS), I'm fine if you don't want to capture the pattern list in a variable
```
      FILES := $(filter %$(SHARED_LIBRARY_SUFFIX) %.txt, $(call FindFiles, $(OPENJCEPLUS_OCK_DIR))), \
```
but I will insist that the filter not admit any files that would only be relevant on a different platform.

I believe it was a mistake not to use `SHARED_LIBRARY_SUFFIX` originally in https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/710; let's fix that now.

Reply:
Replaced with SHARED_LIBRARY_SUFFIX and removed .x, as Z/OS does not use the open-source OpenJCEPlus and has a different build process.